src/main/webapp/WEB-INF/view/feed.jsp

```diff
@@ -1,10 +1,47 @@
 <!DOCTYPE html>
 <html>
 <head>
- <title>Activity Feed</title>
+	<title><%= "Activity Feed" %></title>
+	<link rel="stylesheet" href="/css/main.css" type="text/css">
+	<style>
+	    #feed {
+	      background-color: white;
+	      height: 500px;
+	      overflow-y: scroll
+	    }
+  	</style>
```

Comment:
In my experience, something like this would go in a stylesheet. I think the other style classes are in the stylesheet linked in line 5.